config/jest/jest.config.js

```diff
@@ -22,5 +22,6 @@ module.exports = {
   setupFiles: [
     '<rootDir>/config/jest/enzyme.config.js'
   ],
-  testEnvironment: '<rootDir>/config/jest/jsdom.config.js'
+  testEnvironment: '<rootDir>/config/jest/jsdom.config.js',
+  testURL: 'http://localhost/'
```

Comment:
This line is used so TravisCI does not complain about `localStorage is not available for opaque origins`

[https://jestjs.io/docs/en/configuration.html#testurl-string](https://jestjs.io/docs/en/configuration.html#testurl-string)
[https://github.com/jsdom/jsdom/issues/2304#issuecomment-408327115](https://github.com/jsdom/jsdom/issues/2304#issuecomment-408327115)